docs/started/daemons.md

```diff
@@ -0,0 +1,91 @@
+
+## FAQ
+
+
+### Conveyor daemons
+It is important to know the following:
+* The throttler daemon will need the preparer to work.
+* The preparer is a daemon that optimizes transfer requests, while recommended to install, it's not mandatory.
+* The submitter is the only daemon needed to submit transfers and can do a subset of what the preparer can do.
+* To update the state of requests, the conveyor poller (polls for changes) or conveyor receiver (listens for changes)  are needed to understand the new state.
+* The finisher analyzes this new state and updates the state.
+
+### What happens when a rule is stuck?
+The judge repairer will analyze why the transfer is stuck and try to unstuck it, eventually resubmitting the request.
+
+### What happens when new data is added to an existing dataset that already has replicas?
+The judge evaluator will keep track of new data added to datasets that are already replicated to trigger the necessary transfer requests to ensure all new data is copied to the RSEs.
+
+### What is the purpose of the minos daemons?
+An human operator can declare some datasets as temporarily unavailable due to maintenance, outages, etc ...The operator will set an expiration time on the temporary unavailable status. When the expiration time is reached, the minos-temporary-expiration will put the replicas back in available state.
+
+### What is the relationship between auditor, rucio-dumper and dark-reaper?
+The dumper will create a dump of all the files in an RSE that will be passed to the auditor. The auditor will check for inconsistencies and mark missing data as dark data (quarantined replicas).
+Dark reaper is the one deleting this dark data to free up space from the quarantined replicas table.
+
+### How is data deleted?
+When replicas are healthy, the judge-cleaner will set a tombstone on replicas where the lifetime has expired. These replicas are taken by the reaper and they are deleted. 
+Sometimes, replicas can become unhealthy. A dump is created by the dumper daemon. The auditor checks these dumps and declares replicas 
+
+## How is a replica declared bad?
+- An operator can declare a replica as bad issuing rucio cli commands. These bad replicas are taken by the necromancer daemon and then deleted if they cannot be recovered from other RSEs.
+- the suspicios-replica-recoverer is a daemon that will analyze different counters (transfer errors, download errors, etc ...) to mark replicas as being suspicious to have an issue. After hitting
+a certain limit (configurable), the replica is marked as bad and eventually consumed by the necromancer.
+
+## What is the purpose of undertaker?
+A dataset is "never" deleted, however, when the dataset is known to be spurious, there is no point having it in the catalog. The undertaker daemon takes care to remove these datasets.
```

Comment:
"spurious" is an odd worse choice, I'd just call it "bad" like the status defines it.